lidiabrowser/sync/populate.py

```diff
@@ -0,0 +1,210 @@
+from typing import Optional
+from django.db import transaction
+
+import yaml
+import logging
+
+import sync.models as syncmodels
+from sync.zoteroutils import get_attachment_url, get_attachment_id_from_url
+from lidia.models import ArticleTerm, BaseAnnotation, Category, ContinuationAnnotation, LidiaTerm, Publication as LidiaPublication, TermGroup
+from lidia.models import Annotation as LidiaAnnotation
+from lidia.models import Language  #, ArticleTerm, LidiaTerm, Category
```

Comment:
`Publication as LidiaPublication` can go now that you use `syncmodels`, also old placeholder comment `#, ArticleTerm, LidiaTerm, Category`